tracks/ruby/exercises/word-count/mentoring.md

```diff
@@ -25,37 +25,35 @@ end
 
 ```
 
-An exceptionally nice solution uses `group_by(&:itself)`, and the `transform_values` method introduced in Ruby 2.4
-
-```ruby
-def word_count
-  words.group_by(&:itself).transform_values { |v| v.count }  
-end
-
-```
-which can be even more elegant with symbol-to-proc: `transform_values(&:count)` 
+### Reasonable Variants
+- Using `reduce`/`inject` for the counting.
 
+- An exceptionally nice solution uses `group_by(&:itself)`, and the `transform_values` method introduced in Ruby 2.4
 
+  ```ruby
+    def word_count
+      words.group_by(&:itself).transform_values { |v| v.count }  
+    end
+  ```
+  which can be even more elegant with `Symbol#to_proc`: `transform_values(&:count)` 
 
-### Common suggestions 
-- [Minimal Viable Solution] For the counting, solutions with `each_with_object`, `reduce`/`inject` and `group_by` are acceptable approaches. 
-Iterations with `each`, `for` or others with separate counters are not.
+### General
+- At this point in the track, students have had multiple opportunities to practive with avoiding the accumulator pattern (with a counter variable). 
+- Solutions with `each`, `for` or others are not considered approvable.
 - The RegEx gets complicated in the last test, where it should catch both `"large"` and `'large'` ` => { large: 2 }`. 
-As long as they got something that catches a word with or without `'`, you can give away the last step for free.  
+As long as they got something that catches a word with or without `'`, it's fine to give away the last step for free.  
 http://www.rubular.com/ 
-- [Minimal Viable Solution] Extracting the preparation of the input into a separate method.
-- One strategy people use to catch quoted words is an intermediate step like 'strip_quotes'. 
-Suggest to solve it within the RegEx. 
 
 ### Talking points
-- Why the above mentioned solutions are preferred over iterations with separate counters.
+
+- Extracting the preparation of the input into a separate (private) method.
+- `intermediate steps to strip quotes`: One strategy people use to catch quoted words is an intermediate step like 'strip_quotes'. Suggest to solve it within the RegEx. 
 - Storing the RegEx in a Constant.
-- Public and Private Interface.
 - The goal of separating the preparing of the input from the processing itself.
 - Do's and don'ts in initializers.  
-- Making RegEx's readable.
 
-### Passing RegEx's
+### Mentor Research 
```

Comment:
It's still not 100% obvious to me what the goal of this section is when I look at "Mentor Research". I think I understood after you explained it when you first introduced it, and I _kinda_ get it now too, but maybe there's a name that makes this clearer? Just some thoughts, we don't need to change anything.

Reply:
It's meant to collect the research that mentors encounter while mentoring this exercise. So that other mentors don't have to do the same research. 
The passing regex'es is not the best example of why this research is useful, but I hope that other mentors will add their research notes as well.
⚡️:  Research Notes
Is Research Notes a better title?

Reply:
I like that 👍

Reply:
Should we point out that mentors can / are expected to add their own research notes in this section? (that's what's intended, right?)

So maybe `### Research Notes (add your own on Github)` or something?

Reply:
Hmm, I don't think so. At the bottom of the Mentor Notes is says something like: anything to add? with a link to this repo. 
If we add it here, it looks like the rest of the Notes are not expecting changes. 
I do understand your point, but I think we'll need to address that in a different way.